jsonrpc/src/methods/beacon.json

```diff
@@ -157,7 +157,10 @@
     ],
     "result": {
       "$ref": "#/components/contentDescriptors/TraceRecursiveFindContentResult"
-    }
+    },
+    "errors":[{
+      "$ref": "#/components/errors/ContentNotFoundErrorWithTrace"
+    }]
   },
```

Comment:
Seems like we're also going to want to add this error to simple `RecursiveFindContent` requests? I'm split on whether we want to include the trace w/ the error for these endpoints...